Sming/Libraries/google-cast/samples/basic/component.mk

```diff
@@ -1,4 +1,4 @@
-COMPONENT_DEPENDS := google-cast 
```

Comment:
The Google-cast component should depend itself on MDNS. The sample should depend only on google-cast. And the magical build and dependency system will figure out the rest. If I am right, just revert this line.